src/_main_/settings.py

```diff
@@ -27,8 +27,8 @@
 # ********  LOAD CONFIG DATA ***********#
```

Comment:
Please avoid committing setting.py unless you have actual changes to it, not just for testing

Reply:
sure